src/main/java/echoServer/Server.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package echoserver;
+
+//import java.util.Scanner;
+
+public class Server {
+    private ServerSocketWrapperInterface socketWrapper;
```

Comment:
I'm confused as to why there are two mocks for ServerSocketWrapperInterface. I'd think that the reason you two made a mock in the first place is to test this Server.java class. However, it looks like you have a test for each mock? Are those two mock classes supposed to be used in production for something?